src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -514,14 +534,30 @@ private static void getFormat(StreamType streamType, DLNAMediaInfo media, DLNAMe
 			format = FormatConfiguration.MP4;
 		} else if (value.contains("mjpg") || value.contains("m-jpeg")) {
 			format = FormatConfiguration.MJPEG;
-		} else if (value.contains("div") || value.contains("dx")) {
-			format = FormatConfiguration.DIVX;
+		} else if (
+				value.startsWith("div") ||
+				value.equals("dx50") ||
+				value.equals("dvx1")
+			) {
+				format = FormatConfiguration.DIVX;
+		} else if (value.startsWith("indeo")) { // Intel Indeo Video: IV31, IV32, IV41 and IV50
+			format = FormatConfiguration.INDEO;
+		} else if (value.equals("yuv")) {
+			format = FormatConfiguration.YUV;
+		} else if (value.equals("rgb") || value.equals("rgba")) {
```

Comment:
"perfect" ? Are you going well ? sick or depressive ? :rofl: 
I'm kidding you because i never read you say that, and i still cannot imagine that you could ever say that. It's anyway good to take :smile: 

Amending on such specific little subject as RGB or YUV is fine, IMO, tough reviewing get uneasy if you don't tell the others that you changed your code.
So after amending, i can apologize as well :wink: